docs/index.md

```diff
@@ -37,7 +37,7 @@ provider "snowflake" {
 
 ## Configuration Schema
 
-**Warning: these values are passed directly to the gosnowflake libarary, which may not work exactly the way you expect. See the [gosnowflake docs](https://godoc.org/github.com/snowflakedb/gosnowflake#hdr-Connection_Parameters) for more.**
+**Warning: these values are passed directly to the gosnowflake library, which may not work exactly the way you expect. See the [gosnowflake docs](https://godoc.org/github.com/snowflakedb/gosnowflake#hdr-Connection_Parameters) for more.**
```

Comment:
You'll have to update this file: https://github.com/chanzuckerberg/terraform-provider-snowflake/blob/main/templates/index.md.tmpl
and then run `make docs`